scripts/ci/jobs/jobs.yml

```diff
@@ -491,6 +496,22 @@
       - docker-credentials
       - skydive-wrappers
 
+- job:
```

Comment:
Why an other job ? We already build a docker image with the tag "latest" for each commit. So we should just push the latest image to both : `skydive/skydive-snapshot:${sha1}` and `skydive/skydive:latest`

Reply:
ok - extended existing job